pkg/util/util.go

```diff
@@ -2,6 +2,9 @@ package util
 
 import (
 	"context"
+	"fmt"
+	"math/rand"
```

Comment:
I think it would be better to use crypto/rand here if our aim is to anonymize something so that AI could not reverse engineer :P even seeding math/rand could be predictable